jest-integration.config.js

```diff
@@ -23,10 +23,10 @@ module.exports = {
     },
     // We need a high coverage for the HathorWallet class
     './src/new/wallet.js': {
-      statements: 93,
```

Comment:
Why did you reduce the coverage instead of adding tests?

I know that you've added some but can't we continue with the previous coverage and just add more tests for more cases?